src/driver/franka_plan_runner.cc

```diff
@@ -848,6 +851,23 @@ franka::JointPositions FrankaPlanRunner::JointPositionCallback(
   }
   output_to_franka = utils::EigenToArray(output_to_franka_eigen);
 
+  if (plan_exec_opt_ == robot_msgs::plan_exec_opts_t::MOVE_UNTIL_STOP) {
+    Eigen::Map<const Eigen::Matrix<double, 6, 1>> cartesian_contact(
+        robot_state.cartesian_contact.data());
```

Comment:
What happens if we set `MOVE_UNTIL_STOP` but reach the end of a plan without hitting contact?

Reply:
It runs like a normal plan and sends a PLAN_COMPLETE

Reply:
It is on the user to send a plan that goes past the expected contact point

Reply:
Please add a short comment explaining this!

Reply:
Added comment